server/service/microsoft_mdm.go

```diff
@@ -1165,34 +1165,7 @@ func (svc *Service) GetMDMWindowsManagementResponse(ctx context.Context, reqSync
 
 // GetMDMWindowsTOSContent returns valid TOC content
 func (svc *Service) GetMDMWindowsTOSContent(ctx context.Context, redirectUri string, reqID string) (string, error) {
-	tmpl, err := template.New("").Parse(`
-	<html>
-	<head>
-	<style>
-	  button {
-		background-color: #008CBA;
-		color: white;
-		padding: 10px 60px;
-		border: none;
-		cursor: pointer;
-	  }
-	</style>
-	</head>
-	<body>
-	  <center>
-		<img src="https://fleetdm.com/images/logo-blue-162x92@2x.png">
-		<br>
-		<h2>Terms and conditions</h2>
-		<br> Terms and Conditions PDF content should go here <center>
-		  <br>
-		  <button type="button" onClick="acceptBtn()">Accept</button>
-		  <script>
-			function acceptBtn() {
-			  window.location = "{{.RedirectURL}}" + "?IsAccepted=true&OpaqueBlob={{.ClientData}}";
-			}
-		  </script>
-	</body>
-	</html>`)
+	tmpl, err := template.ParseFiles("frontend/templates/windowsTOS.html")
```

Comment:
Do we know if we can use package `html/template` for this rather than `text/template`? 

@roperzh, if I recall correctly, @marcosd4h ran into some problems with `html/template` over-escaping the URLs? Does that sound familiar? Also, where do you think would be the best place to locate the template in the repo? Would it be more at home in `/server` or `/frontend`

Reply:
@roperzh im not sure about @marcosd4h attempts to make this a template but I can try with HTML template and see what happens.

I also think having this HTML file like in the frontend directory makes the most sense as it is part of the frontend of the fleet application. I think we can expand the frontend to be the fleet UI application as well as simpler html pages like this.

Reply:
@gillespi314 @robusto `html/template` seems to work. made the change

Reply:
> I'd argue that its more intuitive that all UI code lives in the frontend directory, rather then some in the frontend, some in the server directories.

That makes sense. From the backend perspective, I was thinking in this case, the file is a server-rendered template and needs to be parsed by Go in order to work as intended. There's a similar concept for the HTML [templates](https://github.com/fleetdm/fleet/tree/main/server/mail/templates) used by the server to render email body content. So that was the directory structure that I had in mind. I'm fine either way for now. If we want to move in the direction of consolidating server-rendered Go templates into the `/frontend` directory, it might make sense to open a ticket to track that proposal. I'm fine either way for now.